CHANGELOG.md

```diff
@@ -11,6 +11,13 @@
 * Adds launch.json option to suppress Just-In-Time compiler optimizations.
 * Due to the previous two items and work from the .NET Team, it is now possible to easily debug into ASP.NET itself in projects running against .NET Core 2.1 preview 1. Support for debugging into all the managed code in .NET Core will come in future .NET Core 2.1 builds. Instructions are in the [wiki](https://github.com/OmniSharp/omnisharp-vscode/wiki/Debugging-into-the-.NET-Framework-itself).
 
+#### Alternative Omnisharp Support
+
+* Adds support to use the "omnisharp.path" option to download alternative copies of OmniSharp. The possible values for this option are:
+    * "latest" - Use the latest CI build
+    * Version to use - The particular OmniSharp version the user desires like "1.29.2-beta.60"
```

Comment:
"To use a specific pre-release build ..."
Describe where users can find these numbers, eg the folder omnisharp gets downloaded to

Reply:
`latest` looks good. for the last one:
```
* "<version>" - Use a specific version of OmniSharp. Example: `1.29.2-beta.60`
```